posthog/management/commands/test_migrations_are_safe.py

```diff
@@ -64,7 +64,10 @@ def validate_migration_sql(sql) -> bool:
                 f"\n\n\033[91mFound a DROP TABLE command. This could lead to unsafe states for the app. Please avoid dropping tables.\nSource: `{operation_sql}`"
             )
             return True
-        if "CONSTRAINT" in operation_sql and (
+
+        # This pattern searches for the word 'CONSTRAINT' that isn't immediately preceded by 'DROP'
+        constraint_pattern = re.compile(r"(?<!DROP )CONSTRAINT")
+        if constraint_pattern.search(operation_sql) and (
```

Comment:
This should allow us to drop constraints but not add new ones, which I'm fairly certain is a quick operation on postgres and won't lock up the table.

Reply:
This is fine